doc/contributing/collaborator-guide.md

```diff
@@ -540,6 +540,9 @@ The TSC serves as the final arbiter where required.
    landing. If you are unsure exactly how to format the commit messages, use
    the commit log as a reference. See [this commit][commit-example] as an
    example.
+6. Ideally, check the recent commits on `main` to evaluate any potential
```

Comment:
```suggestion
6. Ideally, check the recent commits on the base branch to evaluate any potential
```

Reply:
`main` and the base branch in this context are synonym to me, I will treat this as a nit if that's ok

Reply:
Sure! FWIW the base branch refers to whatever branch the PR is targeting, so I think it makes more sense, as the PR could also be targeting `vX.x-staging`, etc.

Reply:
I don't think this is feasible. It's already hard to land something as is.